@amimodel/compileC.m

```diff
@@ -176,35 +80,35 @@ function compileC(this)
         end
     end
     
-    if(this.cfun(1).(this.funs{j}))
-        fprintf(['ffuns | ']);
-        ffuns = cellfun(@(x) fullfile(this.wrap_path,'models',this.modelname,[this.modelname '_' x '.cpp']),this.funs,'UniformOutput',false);
+    
+    funsForRecompile = this.funs(structfun(@(x) logical(x), this.cfun(1)));
+    if(numel(funsForRecompile))
```

Comment:
It is subsetted, it contains only the function names that have to be recompiled. We could sum over cfun, yes, but we need the subset afterwards anyways.